templates/products/products.css

```diff
@@ -1 +1,85 @@
 /* product specific styles */
+
+.block.cards.products {
+    & > ul {
+      list-style: none;
+      padding: 0;
+      overflow: hidden;
+      position: relative;
+      grid-template-columns: repeat(2, 50%);
+      height: auto;
+    }
+
+    & > ul li {
+      position: relative;
+      display: block;
+      background: #eee;
+      overflow: hidden;
+      text-transform: uppercase;
+      color: #00b0f0;
+      font-size: 11px;
+      font-weight: bold;
+      cursor: hand;
+      cursor: pointer;
+    }
+
+    & > ul li img {
+      width: auto;
+      height: auto;
+    }
+
+    & .cards-card-body {
+        margin: 0;
+        padding: 0;
+        position: absolute;
+        z-index: 2;
+    }
+
+    & .cards-card-body p {
+        margin: 0;
+        padding: 0;
+    }
+
+    & .cards-card-body .title {
+        padding: 5px;
+        text-decoration: underline;
+    }
```

Comment:
Nesting look good.  Note, you can nest child elements too.  .cards-card-body could look like this.

```
.cards-card-body {
    margin: 0;
    padding: 0;
    position: absolute;
    z-index: 2;

    p {
      margin: 0;
      padding: 0;
    }

    .title {
      padding: 5px;
      text-decoration: underline;
    }
  }